ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/TransactionsReader.scala

```diff
@@ -446,3 +523,35 @@ private[appendonlydao] final class TransactionsReader(
     mat
   }
 }
+
+private[appendonlydao] object TransactionsReader {
+  private[appendonlydao] def splitRange(
```

Comment:
What is this method used for?

What can a caller expect?

Reply:
Done, added a comment